scripts/build/custom.ts

```diff
@@ -0,0 +1,34 @@
+import {exec} from "child_process"
+import {BundleDef} from "./bundles"
+import chalk from "chalk"
+import ora from "ora"
+
+function asyncExecuteCommand(command: string) {
+  return new Promise((resolve, reject) =>
+    exec(command, (error, stdout, stderr) => {
+      stdout = stdout.trim()
+      stderr = stderr.trim()
+
+      if (error) {
+        reject({stdout, stderr})
+        return
+      }
+      resolve({stdout, stderr})
+    })
+  )
+}
+
+export async function customBuilder(bundle: BundleDef) {
```

Comment:
Why not just use `export default async function` like all other builders?

Reply:
not in my habit, default export/import is evil